src/EditorFeatures/CSharpTest/StringCopyPaste/PasteUnknownSourceIntoMultiLineInterpolatedRawStringTests.cs

```diff
@@ -707,7 +707,7 @@ public void TestQuotesIntoHeader2()
     |}""""""",
 @"var x = $""""""""
     """"""
-    ""[||]""""""",
+    [||]""""""""",
```

Comment:
this is an improvement over the prior behavior.  the caret now stays before the delimiters instead of moving partway into the closing delimiters.